content/applications/finance/accounting/others/multicurrencies.rst

```diff
@@ -7,6 +7,4 @@ Multicurrency
 .. toctree::
    :titlesonly:
 
-   multicurrencies/how_it_works
-   multicurrencies/invoices_payments
-   multicurrencies/exchange
\ No newline at end of file
+   multicurrencies/multicurrency
```

Comment:
Done :)

Reply:
As explained live, the toctree doesn't look great when you mix pages containing only other toctrees with pages directly containing content.
The page 'Multicurrency System' kind of makes you think it's a failed bullet point under 'Adviser'. However, it's a page at the same level as 'Adviser'.
![chrome_E83TJfYfJT](https://user-images.githubusercontent.com/70884563/184164519-b232512f-648f-47c1-aa96-71ea2d754a40.png)
So, on the `content/applications/finance/accounting/others.rst` move the `others/multicurrencies`  on top of the pag's toctree so it appears first, similarly to the first two pages mentionned under 'Getting Started'
![chrome_jvU2C9zUiI](https://user-images.githubusercontent.com/70884563/184166602-6bec4eff-c4f5-4398-928a-844673d25fb6.png)
It's a quick fix, but maybe we should increase the spacing to avoid this issue in the future. @AntoineVDV

Reply:
Good catch

Reply:
> It's a quick fix, but maybe we should increase the spacing to avoid this issue in the future. @AntoineVDV

@xpl-odoo Please create a task in the "Documentation" project and tag me on it :) This should be part of a larger batch of changes to the display of toctree items.